packages/blocks/src/page-block/edgeless/frame-resize-observer.ts

```diff
@@ -14,9 +14,10 @@ export class FrameResizeObserver {
    * So we need to cache observed element.
    */
   private _cachedElements = new Map<string, Element>();
+  private _lastRect = new Map<string, DOMRectReadOnly>();
```

Comment:
`lastRects` aligns sementic better (with "s" that indicates there can be multiple objects). Also, is that somewhere required to cleanup this map?

Reply:
yeah, we need to cleanup this map.